test/TestProjects/Accessibility-LowLevel/Generated/AccessibilityClient.cs

```diff
@@ -147,22 +146,10 @@ internal HttpMessage CreateOperationInternalRequest(RequestContent content, Requ
             request.Uri = uri;
             request.Headers.Add("Content-Type", "application/json");
             request.Content = content;
-            message.ResponseClassifier = ResponseClassifier200.Instance;
             return message;
         }
 
-        private sealed class ResponseClassifier200 : ResponseClassifier
-        {
-            private static ResponseClassifier _instance;
-            public static ResponseClassifier Instance => _instance ??= new ResponseClassifier200();
-            public override bool IsErrorResponse(HttpMessage message)
-            {
-                return message.Response.Status switch
-                {
-                    200 => false,
-                    _ => true
-                };
-            }
-        }
+        private static ResponseClassifier _responseClassifier200;
+        private static ResponseClassifier ResponseClassifier200 => _responseClassifier200 ??= new CoreResponseClassifier(stackalloc int[] { 200 });
```

Comment:
[Here](https://github.com/Azure/autorest.csharp/blob/ebc6eede5294c2b0300bf86b28e852c5fa3a4f03/test/TestProjects/Accessibility-LowLevel/Generated/AccessibilityClient.cs#L152-L153) is the generated code for `ResponseClassifier200` bases on https://github.com/Azure/autorest.csharp/issues/2006#issuecomment-1055883992